src/Servers/Kestrel/Core/src/CoreStrings.resx

```diff
@@ -148,7 +148,7 @@
     <value>Invalid Host header: '{detail}'</value>
   </data>
   <data name="BadRequest_InvalidRequestHeadersNoCRLF" xml:space="preserve">
-    <value>Invalid request headers: missing final CRLF in header fields.</value>
+    <value>Invalid CR before CRLF to complete the header.</value>
```

Comment:
Is this the right message here? Looks like we also use this for cases like:

`"Header-1: value1\r\nHeader-2: value2\r\n\r "`